app/services/jobseekers/job_applications/quick_apply.rb

```diff
@@ -28,7 +28,6 @@ def recent_job_application
   def attributes_to_copy
     Jobseekers::JobApplication::PersonalDetailsForm.fields +
       Jobseekers::JobApplication::ProfessionalStatusForm.fields +
-      Jobseekers::JobApplication::EmploymentHistoryForm.fields +
```

Comment:
This is an empty array. This form is the form used on the employment history main step (the form which contains only a 'continue' button and no fields), rather than the form used to create an individual employment history record (which is EmploymentForm).